src/app/garfile/arrival/post.controller.js

```diff
@@ -122,58 +100,30 @@ module.exports = async (req, res) => {
   // Define voyage
   const voyage = req.body;
   delete voyage.buttonClicked;
-  if (voyage.portChoice === 'No') {
-    logger.debug('Testing arrival Lat and Long values...');
-
-    if (voyage.arrivalLatDirection && voyage.arrivalLatDirection.toUpperCase() == 'S'){
-      const convertedLat = parseFloat(voyage.arrivalDegrees) + parseFloat((voyage.arrivalMinutes/60) + parseFloat((voyage.arrivalSeconds/3600).toFixed(6)));
-      voyage.arrivalLat = '-' + parseFloat(convertedLat).toFixed(6);
-    }
-    else{
-      const convertedLat = parseFloat(voyage.arrivalDegrees) + parseFloat((voyage.arrivalMinutes/60) + parseFloat((voyage.arrivalSeconds/3600).toFixed(6)));
-      voyage.arrivalLat = parseFloat(convertedLat).toFixed(6);
-    }
-    
-    if (voyage.arrivalLongDirection && voyage.arrivalLongDirection.toUpperCase() == 'W'){
-      const convertedLong = parseFloat(voyage.arrivalLongDegrees) + parseFloat((voyage.arrivalLongMinutes/60) + parseFloat((voyage.arrivalLongSeconds/3600).toFixed(6)));
-      voyage.arrivalLong = '-' + parseFloat(convertedLong).toFixed(6);
-    }
-    else{
-      const convertedLong = parseFloat(voyage.arrivalLongDegrees) + parseFloat((voyage.arrivalLongMinutes/60) + parseFloat((voyage.arrivalLongSeconds/3600).toFixed(6)));
-      voyage.arrivalLong = parseFloat(convertedLong).toFixed(6);
-    }
-
-    logger.debug(voyage.arrivalLat);
-    logger.debug(voyage.arrivalLong);
-    const combinedCoordinates = `${voyage.arrivalLat.toString()} ${voyage.arrivalLong.toString()}`;
-    voyage.arrivalPort = combinedCoordinates;
-    logger.debug(voyage.arrivalPort);
-  } else {
-    // If 'Yes' is selected then clear the coordinate values
+
+  if (voyage.portChoice === 'Yes') {
     voyage.arrivalLat = '';
```

Comment:
Question: What is the reason why `arrivalLat`/`arrivalLong` is `''` (and in other places is `undefined` or `null`?
*  Note, I will accept their existance.

Reply:
Explanation give, see the below:
```
only one I didn't address was where I dynamically added lat/long fields to Cookie object so could be absent depending on logical route.
I think that is not a problem as no code assumed them to be present and if they are assumed to be a string then coalesces to make sure no error.
```